public/locales/az/gamepage.json

```diff
@@ -94,13 +97,22 @@
             "start": "İndi oyna",
             "stop": "Oynanır (Dayandır)"
         },
+        "queue": {
+            "remove": "Remove from Queue"
```

Comment:
We have this same string in 2 places: `button.queue.remove` and `label.queue.remove`, maybe we can unify them?